mvnsettingsUseEnvVars.xml

```diff
@@ -0,0 +1,24 @@
+<settings>
```

Comment:
This one doesn't work right? I suggest we attach it do the ticket but not bother checking in to keep things simple.

Reply:
Done, with comment on ticket as to purpose of file.